quantum/keymap_extras/keymap_dvorak_fr.h

```diff
@@ -0,0 +1,131 @@
+ * └────┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴──────────┘
+ */
+// Row 1
+#define DF_LDAQ KC_GRV  // «
+#define DF_RDAQ KC_1    // »
+#define DF_SLSH KC_2    // /
+#define DF_MINS KC_3    // -
+#define DF_EGRV KC_4    // è
+#define DF_BSLS KC_5    // (backslash)
+#define DF_CIRC KC_6    // ^ (dead)
+#define DF_LPRN KC_7    // (
+#define DF_GRV  KC_8    // ` (dead)
+#define DF_RPRN KC_9    // )
+#define DF_UNDS KC_0    // _
+#define DF_LBRC KC_MINS // [
+#define DF_RBRC KC_EQL  // ] (dead)
+// Row 2
+#define DF_COLN KC_Q    // :
+#define DF_QUOT KC_W    // '
+#define DF_EACU KC_E    // é
+#define DF_G    KC_R    // G
+#define DF_DOT  KC_T    // .
+#define DF_H    KC_Y    // H
+#define DF_V    KC_U    // V
+#define DF_C    KC_I    // C
+#define DF_M    KC_O    // M
+#define DF_K    KC_P    // K
+#define DF_Z    KC_LBRC // Z
+#define DF_DIAE KC_RBRC // ¨ (dead)
+// Row 3
+#define DF_O    KC_A    // O
+#define DF_A    KC_S    // A
+#define DF_U    KC_D    // U
+#define DF_E    KC_F    // E
+#define DF_B    KC_G    // B
+#define DF_F    KC_H    // F
+#define DF_S    KC_J    // S
+#define DF_T    KC_K    // T
+#define DF_N    KC_L    // N
+#define DF_D    KC_SCLN // D
+#define DF_W    KC_QUOT // W
+#define DF_TILD KC_NUHS // ~ (dead)
+// Row 4
+#define DF_AGRV KC_NUBS // à
+#define DF_SCLN KC_Z    // ;
+#define DF_Q    KC_X    // Q
+#define DF_COMM KC_C    // ,
+#define DF_I    KC_V    // I
+#define DF_Y    KC_B    // Y
+#define DF_X    KC_N    // X
+#define DF_R    KC_M    // R
+#define DF_L    KC_COMM // L
+#define DF_P    KC_DOT  // P
+#define DF_J    KC_SLSH // J
+
+/* Shifted symbols
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ * │ 1 │ 2 │ 3 │ 4 │ 5 │ 6 │ 7 │ 8 │ 0 │ 0 │ + │ % │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │ ? │ < │ > │   │ ! │   │   │   │   │   │   │ = │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │   │   │   │   │   │   │   │   │   │   │   │ # │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │    │ ç │ | │   │ @ │   │   │   │   │   │   │   │          │
+ * └────┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴──────────┘
+ */
+// Row 1
+#define DF_ASTR S(DF_LDAQ) // *
+#define DF_1    S(DF_RDAQ) // 1
+#define DF_2    S(DF_SLSH) // 2
+#define DF_3    S(DF_MINS) // 3
+#define DF_4    S(DF_EGRV) // 4
+#define DF_5    S(DF_BSLS) // 5
+#define DF_6    S(DF_CIRC) // 6
+#define DF_7    S(DF_LPRN) // 7
+#define DF_8    S(DF_GRV)  // 8
+#define DF_9    S(DF_RPRN) // 9
+#define DF_0    S(DF_UNDS) // 0
+#define DF_PLUS S(DF_LBRC) // +
+#define DF_PERC S(DF_RBRC) // %
+// Row 2
+#define DF_QUES S(DF_COLN) // ?
+#define DF_LABK S(DF_QUOT) // <
+#define DF_RABK S(DF_EACU) // >
+#define DF_EXLM S(DF_DOT)  // !
+#define DF_EQL  S(DF_DIAE) // =
+// Row 3
+#define DF_HASH S(DF_TILD) // #
+// Row 4
+#define DF_CCED S(DF_AGRV) // ç
+#define DF_PIPE S(DF_SCLN) // |
+#define DF_AT   S(DF_COMM) // @
```

Comment:
```suggestion
#define DV_ASTR S(DV_LDAQ) // *
#define DV_1    S(DV_RDAQ) // 1
#define DV_2    S(DV_SLSH) // 2
#define DV_3    S(DV_MINS) // 3
#define DV_4    S(DV_EGRV) // 4
#define DV_5    S(DV_BSLS) // 5
#define DV_6    S(DV_CIRC) // 6
#define DV_7    S(DV_LPRN) // 7
#define DV_8    S(DV_GRV)  // 8
#define DV_9    S(DV_RPRN) // 9
#define DV_0    S(DV_UNDS) // 0
#define DV_PLUS S(DV_LBRC) // +
#define DV_PERC S(DV_RBRC) // %
// Row 2
#define DV_QUES S(DV_COLN) // ?
#define DV_LABK S(DV_QUOT) // <
#define DV_RABK S(DV_EACU) // >
#define DV_EXLM S(DV_DOT)  // !
#define DV_EQL  S(DV_DIAE) // =
// Row 3
#define DV_HASH S(DV_TILD) // #
// Row 4
#define DV_CCED S(DV_AGRV) // ç
#define DV_PIPE S(DV_SCLN) // |
#define DV_AT   S(DV_COMM) // @
```

Reply:
I'm not sure about this changes. It will conflict with dvorak if someone wants to use both layouts on a board